policies/templates/gcp_bigquery_cmek_encryption_v1.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-bq-cmek-encryption-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPBigqueryCMEKEncryptionConstraintV1
```

Comment:
nit: "BigQuery" to be consistent with the product naming (https://cloud.google.com/bigquery/)

Reply:
Fixed.